src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriverExtension.java

```diff
@@ -72,7 +81,7 @@ public T create(
 			final int batchSize
 	) throws OmgShootMyFootException, InterruptedException {
 		return (T) new com.emc.mongoose.storage.driver.pravega.PravegaStorageDriver<I, O>(
-				Constants.DEFAULT_CONTROLLER_URI, stepId, dataInput, storageConfig, verifyFlag, batchSize
```

Comment:
the URI schema value should be under the "storageConfig" structure with the relative path "net-uri-schema", no additional argument is needed